packages/docusaurus-theme-live-codeblock/src/theme/Playground/index.js

```diff
@@ -11,13 +11,55 @@ import clsx from 'clsx';
 
 import styles from './styles.module.css';
 
-function Playground({children, theme, transformCode, ...props}) {
```

Comment:
Removed `transformCode` prop. Never coming down from parent components.
Also, got read of `...props` because:
1. this component is not written in Typescript, so it's hard to know what's inside ...props
2. other keys that exist in `props` are `live` and `metastring`, which are not used here. Therefore excluding them.